spec/ruby/core/thread/thread_variables_spec.rb

```diff
@@ -13,12 +13,12 @@
     @t.thread_variable_set :a, 2
     @t.thread_variable_set :b, 4
     @t.thread_variable_set :c, 6
-    @t.thread_variables.sort.should == [:a, :b, :c]
+    @t.thread_variables.should include(:a, :b, :c)
```

Comment:
Did those fail actually? They should not since they are thread_variables of a new Thread not doing any IO.

Reply:
They didn't fail, but since the other one failed I took the opportunity to make both work independent of any other variables being set.

Reply:
Could you restore this spec as it was?